src/Linux.php

```diff
@@ -22,6 +22,17 @@ public function get($id)
         return $this->parseProcessData($out[0]);
     }
 
+    /**
+     * Check whether the process identified by ID exists.
+     * @param int
+     * @return bool
+     */
+    public function exists($id)
+    {
+        $out = $this->runCommand("ps u -p ". intval($id));
+        return (is_array($out) && count($out)>0) ? true : false;

```

Comment:
maybe you can already do:
`return (is_array($out) && count($out) > 0);`
?
(also include spacing between `>` operator --> = ) > 0  :p

Reply:
Fixed